ops/daily-backup.sh

```
@@ -15,9 +16,29 @@ IGOR_TMP=""
+    if [[ -r "${guard_path}" ]]; then
+      # shellcheck source=/dev/null
+      source "${guard_path}"
+      return
+    fi
+  done
+
+  log "ERROR: required backup guard library missing"
+  exit 1
+}
+
+BACKUP_GUARD_CONTEXT="daily"
```

Shellcheck (0.11.0)

[warning] BACKUP_GUARD_CONTEXT appears unused. Verify use (or export if used externally).

(SC2034)